src/Assistants/InvoiceAssistant.php

```diff
@@ -0,0 +1,406 @@
+                                    'defaultValue' => false,
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.invoiceAddressEqualShippingAddress'
+                                    ]
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+                "stepTwo" => [
+                    "title" => "invoiceAssistant.stepTwoTitle",
+                    "sections" => [
+                        [
+                            "title" => 'invoiceAssistant.infoPageTitle',
+                            "form" => [
+                                "info_page_toggle" => [
+                                    'type' => 'toggle',
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.infoPageToggle',
+                                    ]
+                                ],
+                            ],
+                        ],
+                        [
+                            "title" => 'invoiceAssistant.infoPageTypeTitle',
+                            "description" => 'invoiceAssistant.infoPageTypeDescription',
+                            "condition" => 'info_page_toggle',
+                            "form" => [
+                                "info_page_type" => [
+                                    'type' => 'select',
+                                    'defaultValue' => '1',
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.infoPageTypeName',
+                                        'listBoxValues' => [
+                                            [
+                                                "caption" => 'invoiceAssistant.infoPageInternal',
+                                                "value" => '1',
+                                            ],
+                                            [
+                                                "caption" => 'invoiceAssistant.infoPageExternal',
+                                                "value" => '2',
+                                            ],
+                                        ],
+                                    ],
+                                ],
+                            ],
+                        ],
+                        [
+                            "title" => '',
+                            "description" => 'invoiceAssistant.infoPageNameInternal',
+                            "condition" => 'info_page_toggle && info_page_type == 1',
+                            "form" => [
+                                "internal_info_page" => [
+                                    "type" => 'category',
+                                    'defaultValue' => '',
+                                    'isVisible' => "info_page_toggle == true && info_page_type == 1",
```

Comment:
Das `== true` macht keinen Sinn. Und brauchst du das `isVisible`, wenn oben schon das gleiche in `condition` steht?